website/blog/2024-02-29-cortex-slack.md

```diff
@@ -0,0 +1,79 @@
+- Most significantly, we can now **bake reasoning capabilities into our dbt models** by describing what we want to happen.
+
+Analytics Engineers have always existed at the intersection of business context and data - LLMs on the warehouse make it possible to embed more business context _and_ unlock more data, increasing our leverage in both directions at once.
+
+## Anatomy of an LLM-powered workflow
+
+When we were looking for a first AI-powered use case in our analytics stack, we wanted to find something that:
+
+- Solves a real business problem for us today
+- Makes use of the unique capabilities of LLMs
+- Was cognisant of their current uncertainties and limitations
+- Anticipated future improvements to the models available to us, so things that don't work today might soon work very well indeed.
+
+Once we selected our use case, the analytics engineering work of building and orchestrating the new dbt models felt very familiar; in fact it was exactly the same as any other model I've built.
+
+- I still built a DAG in layers, with existing staging models as the foundation and building new modular segments on top
+- I still followed the same best practices and conventions around writing, styling and versioning controlling my code
+- I still ensured my models behaved as I expected by going through a code review and automated testing process, before deploying my LLM workloads to production with the dbt Cloud orchestrator.
+
+In short, the same dbt I know and love, but augmented by the new power that Cortex exposes.
+
+## Developing our first LLM-powered analytics workflow in dbt Cloud
+
+When thinking about a project that would only be possible if we could make sense of a large volume of unstructured text, I pretty quickly realised this could help me keep up to date with the dbt Community Slack. Even though we spend a lot of time in Slack, there's hundreds of threads taking place across dozens of channels every day, so we often miss important or interesting conversations.
+
+We already pull Slack data into Snowflake for basic analytics, but having triage agent that could keep a watchful eye over the Slack, and let us know about things we'd otherwise have missed would help the Developer Experience team do a better job of keeping our finger on the pulse of dbt developers' needs.
+
+Once it was finished, it looked like this:
+
+[ screenshot tk].
+
+Up to once a day, we'll get a post in our internal Slack with links to a handful of interesting threads for each person's focus areas and a brief summary of the discussion so far. From there, we can go deeper by diving into the thread ourselves, wherever it happens to take place. While developing this I found multiple threads that I wouldn't have found any other way (which was itself a problem, since my model filters out threads once a dbt Labs employee is participating in it, so I kept losing all my testing data).
```

Comment:
I wonder if we could have a screenshot of this?

Reply:
done